src/scope.rs

```diff
@@ -979,9 +910,9 @@ mod tests {
             })
             .finish();
 
-        let req = TestRequest::with_uri("/app/t1").request();
-        let resp = app.run(req);
-        assert_eq!(resp.as_msg().status(), StatusCode::NOT_FOUND);
+        //let req = TestRequest::with_uri("/app/t1").request();
```

Comment:
Is the PR yet to be finished? Or why this code is left commented out?

Also I see below that there is `TestRequest` with the same uri, but with different assert... which actually seems kinda strange.

Reply:
good catch. difference is tail slash